tests/languages/sas/number_feature.test

```diff
@@ -13,9 +13,9 @@ BadFacex
 	["number", "3.2e10"],
 	["number", "0.4e-8"],
 	["number", "1.4E+2"],
-	["number", "BadFacex"]
+	"\r\nBadFacex"

```

Comment:
Like this, the tests may pass but it doesn't really test anything. Please add one or two __real__ hexadecimal numbers.